substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/debugentry/DirEntry.java

```diff
@@ -37,9 +37,11 @@
  */
 public class DirEntry {
```

Comment:
It seems we could use records for `DirEntry`, `FileEntry`, and `MemberEntry` in the future?

Reply:
I'm not sure what you mean by that. Do you mean outside of the `com.oracle.objectfile` package? They are already used outside of `com.oracle.objectfile.debugentry` as they are referenced by `com.oracle.objectfile.elf.dwarf` classes.